pkg/kritis/apis/kritis/v1beta1/genericattestation.go

```diff
@@ -31,9 +31,14 @@ type GenericAttestationPolicy struct {
 	Spec GenericAttestationPolicySpec `json:"spec"`
 }
 
+type AdmissionWhitelistPatternSpec struct {
```

Comment:
I think it's "allowlist" in Kritis codebase.

Reply:
so we are not adopting the name from BinAuthz?

Reply:
I would guess not, but let's confirm with Aysylu.

Reply:
Let‘s stick with allowlist for now then.